simulations/simul/cna/clone.py

```diff
@@ -0,0 +1,61 @@
+from lib2to3.pgen2.token import OP
```

Comment:
Why is this needed?

Reply:
it's not, thanks for the catch, VS tends to like to add imports when I miswrite stuff